### ISSUE TITLE
Nested style arrays working on Android but not in browser

### ISSUE THREAD

**Issue by gianluca-nitti** [Microsoft/reactxp#179]
I'm not sure if this is a bug or my fault, and I apologize if it's an issue in my code and not in ReactXP.

In the [documentation](https://github.com/Microsoft/reactxp/blob/master/docs/docs/styles.md#combining-styles), I read that
> Within a style array, you can also pass nested arrays. This allows easy manipulation of composite styles.
> ```
> // this.props.style might be undefined, a single style, or a (potentially-nested)
> // array of styles.
> <RX.View style={ [_styles.defaultStyle, this.props.style] } />
> ```

So, considering the following TypeScript code (the actual code in my application is more complex - various styles are loaded from different module files - but this is the smallest example which can reproduce the issue):
```
public render() {
    const style1 = RX.Styles.createTextStyle({backgroundColor: "blue"}); //a style
    const style2 = [RX.Styles.createTextStyle({color: "red"}), RX.Styles.createTextStyle({fontSize: 20})]; //an array containing two styles
    return (<RX.Text style={[style1, style2]}>hello, world</RX.Text>); //this should work according to the docs?
}
```
I get the desired result (red text with font size 20 on blue background) when I build for Android, but not in web browsers (I tried Chrome and Edge) where only the first style (blue background) is applied, as shown in the screenshot:
![reactxpnestedstyles](https://user-images.githubusercontent.com/15270303/28465739-1c65e800-6e2a-11e7-8e14-80fe4bce04eb.PNG)

At the moment, as a workaround, I'm concatenating the arrays:
```
return (<RX.Text style={style2.concat(style1)}>hello, world</RX.Text>);
```
which works correctly both on native (tested on Android) and browsers.

**Comment by erictraut** [comment]
Which version of ReactXP are you using? We just added support for nested styles on web last week. Please take a look at your package.json and make sure that you're using version "0.42.0_rc.20".

**Comment by gianluca-nitti** [comment]
Thank you very much and sorry for the dumb question. Yes, i was on an older version (0.42.0_rc.9) and the update fixed the issue.
I tried to update using `npm update` before opening the issue but it didn't update reactxp (maybe npm doesn't care about release candidate numbers, I have to "revise" a bit the semver specification), now I manually changed the version number as you suggested then ran `npm install` and the problem is resolved.
